src/lib/y2storage/proposal/partition_creator.rb

```diff
@@ -157,7 +157,7 @@ def create_partition(vol, partition_id, free_space, primary)
         region = new_region_with_size(free_space.slot, vol.disk_size)
         partition = ptable.create_partition(dev_name, region, partition_type)
         partition.id = partition_id
-        partition.boot = !!vol.bootable
+        partition.boot = !!vol.bootable if ptable.partition_boot_flag_supported?
```

Comment:
`!!variable` is supposed to recover from `variable.nil?`, right? is it a good idea not to do that in that special case?